examples/sklernex/dbscan.py

```diff
@@ -0,0 +1,43 @@
+#===============================================================================
+# Copyright 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#===============================================================================
+
+import numpy as np
+# calling scikit-learn patch
+from sklearnex import patch_sklearn
+patch_sklearn()
+
+from sklearn.datasets import make_blobs
+from sklearn.preprocessing import StandardScaler
+
+centers = [[1, 1], [-1, -1], [1, -1]]
+X, labels_true = make_blobs(
+    n_samples=750, centers=centers, cluster_std=0.4, random_state=0
+)
+
+X = StandardScaler().fit_transform(X)
+
+from sklearn.cluster import DBSCAN
+from sklearn import metrics
+
+db = DBSCAN(eps=0.3, min_samples=10).fit(X)
+labels = db.labels_
+
+# Number of clusters in labels, ignoring noise if present.
+n_clusters_ = len(set(labels)) - (1 if -1 in labels else 0)
+n_noise_ = list(labels).count(-1)
+
+print("Estimated number of clusters: %d" % n_clusters_)
+print("Estimated number of noise points: %d" % n_noise_)
```

Comment:
For many reasons it is better use def main(). Also it would be consistent with other examples.
 Something like this.
```python
def main():
    # my code here

if __name__ == "__main__":
    main()

```

Reply:
I think about scikit-learn examples more as clean way for showcase usage and not use them for validation, although they would be added to testing. 

Also aligning to excising daal4py examples is not necessary the goal - in case comparing to scikit-learn examples they are written in plain form